lib/filesystem.js

```diff
@@ -73,22 +73,35 @@ class Filesystem {
       return callback()
     }
 
-    const tr = options.transform && options.transform(p)
-    if (tr) {
-      return tmp.file(function (err, path) {
-        if (err) { return handler() }
-        const out = fs.createWriteStream(path)
-        const stream = fs.createReadStream(p)
-
-        stream.pipe(tr).pipe(out)
-        return tr.on('end', function () {
-          file.transformed = {
-            path,
-            stat: fs.lstatSync(path)
-          }
+    if (options.transform instanceof Function) {
+      options.transform = [options.transform]
+    }
+
+    if (options.transform instanceof Array && options.transform.length) {
+      options.transform = options.transform.filter(() => true)
```

Comment:
@benlancaster Would be cool if this is something new but as far as I can tell this is a noop call that simply clones the array.  Can you explain exactly what this fixes / handles?